src/bugreporter/odemis_bugreporter.py

```diff
@@ -390,6 +403,21 @@ def compress_files(self):
             logging.exception("Failed to store bug report")
             raise
 
+    def _compress_user_attachments(self) -> None:
+        """Add the user attachments to the zip file"""
+
+        files = self.user_attachments
+
+        if not files:
+            return
+
+        # QUERY: upload the attachments as a seprarate zip?
```

Comment:
Who is this question addressed to? Should we answer it before merging this PR? 

Let's see...
Advantages for a single file:
 * It's easier (for the support team) to download all the data
Advantages for multiple files
 * Maybe it'd be clearer (for the support team) that the user did attach extra files
 * There is no risk of file name conflicts (eg, the user uploads a file named "odemis.log")
 * The attachment size limit is per file (I *think*), so would allow to send slightly bigger attachments.

=> I'd recommend a separate zip file.

Reply:
a single extra attachments.zip with a max size of 100MB or multiple attachments_1.zip, attachments_2.zip each with max size 100MB?

Reply:
I'd say it's fine to have a single file for all of them. Especially, the `_validate_user_attachment_size()` function counts all the attachments together already.